<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dormitory_Info_Admin</class>
 <widget class="QMainWindow" name="Dormitory_Info_Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="ShowResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>691</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>宿舍信息管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowAllBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示全表</string>
    </property>
   </widget>
   <widget class="QComboBox" name="QueryChoice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>按宿舍号</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="QueryEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="QueryBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
